modules/calcite/src/test/java/org/apache/ignite/internal/processors/query/calcite/integration/QueryMetadataIntegrationTest.java

```diff
@@ -42,13 +42,14 @@ public class QueryMetadataIntegrationTest extends AbstractBasicIntegrationTest {
     @Test
     public void testJoin() throws Exception {
         executeSql("CREATE TABLE tbl1 (id DECIMAL(10, 2), val VARCHAR, val2 BIGINT, ts TIMESTAMP(14), PRIMARY KEY(id, val))");
-        executeSql("CREATE TABLE tbl2 (id DECIMAL(10, 2), val VARCHAR, val2 BIGINT, ts TIMESTAMP(14), PRIMARY KEY(id, val))");
+        executeSql("CREATE TABLE tbl2 (id DECIMAL(10, 2) NOT NULL, val VARCHAR, val2 BIGINT, ts TIMESTAMP(14), " +
```

Comment:
Suggestion: + a similar test to cover H2/GridQueryProcessor queries like `SqlFieldsQueryUsageTest`. WDYT?

Reply:
1. `SqlFieldsQueryUsageTest` was created to test DDL when DDL was not supported by calcite, now this test make no sense and can be deleted.
2. I beleive that H2-engine tests for nullability already exists (looks like `IgniteSqlNotNullConstraintTest` checks nullability)